examples/calibration/plot_cutoff_calibration.py

```diff
@@ -0,0 +1,154 @@
+"""
+=======================================================================
+Decision threshold (cutoff point) calibration for binary classification
+=======================================================================
+
+Most machine learning classifiers that offer probability estimation do so by
+optimizing for accuracy (minimizing the classification error). The class with
```

Comment:
Or model likelihood

Reply:
Hey, to get it right; are you suggesting to say something like

Machine learning classifiers often base their predictions on real-valued
decision functions with the threshold set to model the likelihood for each class...

?

Reply:
This comment is merely saying that many models do not use accuracy as their objective. "Likelihood for each class" is not, I think, an appropriate use of "likelihood" as a statistical term.

The key is not what the objective function is (it varies), but that the learning objective often differs from the user's needs. Hence tweaking the threshold can often improve prediction on some extrinsic objective.